pkg/forwarder/forwarder.go

```diff
@@ -163,20 +179,26 @@ func New(cfg Config) (*Worker, error) {
 		if !pool.AppendCertsFromPEM(data) {
 			level.Warn(logger).Log("msg", "no certs found in from-ca-file")
 		}
-		fromTransport.TLSClientConfig.RootCAs = pool
+
+		fromTransport.TLSClientConfig = &tls.Config{
+			Certificates: []tls.Certificate{cert},
+			RootCAs:      pool,
+		}
 	}
-	fromClient := &http.Client{Transport: fromTransport}
 	if cfg.Debug {
+		level.Info(logger).Log("msg", "fromClient transport: Debug ")
 		fromClient.Transport = telemeterhttp.NewDebugRoundTripper(logger, fromClient.Transport)
 	}
 	if len(cfg.FromToken) == 0 && len(cfg.FromTokenFile) > 0 {
+		level.Info(logger).Log("msg", "fromClient transport FromTokenFile")
```

Comment:
Is this log for debugging purposes indicating entry of this code block or for showing the file name of `cfg.FromTokenFile`?

Reply:
In that case we can use `level.Debug()` . 
When debugging we pass `--log-level=debug`, when in normal use it does not show up.

Reply:
same for line 201.
other looks good to me :)

Reply:
done, thanks!